tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -72,27 +72,49 @@ def mysqldump():
     logger.info(f"Complete. {outfile} is {size} bytes.")
 
 
-def mongodump():
+def get_mongodump_command(database=None):
     host = ENV['MONGODB_HOST']
     port = ENV['MONGODB_PORT']
     outdir = MONGODB_DUMPDIR
 
-    logger.info(f"Dumping MongoDB databases on {host}:{port} to {outdir}")
```

Comment:
I think we still might want to include the `host`, `port`, and `outdir` information in the log message.  Right now, unless I am missing something, we only log the database name anymore. What do you think?

Reply:
Makes sense. I added those.